Description: Disable new version check in notices tab
 Debian/Ubuntu specific patch.
Author: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>,
	ChristianB
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1140597

---

Forwarded: not-needed
Reviewed-By: costamagnagianfranco@yahoo.it
Last-Update: <2013-03-07>

Index: boinc/client/client_state.cpp
===================================================================
--- boinc.orig/client/client_state.cpp
+++ boinc/client/client_state.cpp
@@ -653,7 +653,7 @@
     // NOTE: this must be called AFTER
     // read_vc_config_file()
     //
-    newer_version_startup_check();
+//    newer_version_startup_check();
 
     // parse account files again,
     // now that we know the host's venue on each project
Index: boinc/client/scheduler_op.cpp
===================================================================
--- boinc.orig/client/scheduler_op.cpp
+++ boinc/client/scheduler_op.cpp
@@ -138,7 +138,7 @@
         // and project list
         //
         if (!cc_config.no_info_fetch) {
-            gstate.new_version_check();
+//            gstate.new_version_check();
             gstate.all_projects_list_check();
         }
     }
Index: boinc/clientgui/AdvancedFrame.cpp
===================================================================
--- boinc.orig/clientgui/AdvancedFrame.cpp
+++ boinc/clientgui/AdvancedFrame.cpp
@@ -198,7 +198,7 @@
     EVT_MENU(ID_HELPBOINCMANAGER, CAdvancedFrame::OnHelpBOINC)
     EVT_MENU(ID_HELPBOINCWEBSITE, CAdvancedFrame::OnHelpBOINC)
     EVT_MENU(wxID_ABOUT, CAdvancedFrame::OnHelpAbout)
-    EVT_MENU(ID_CHECK_VERSION, CAdvancedFrame::OnCheckVersion)
+    //EVT_MENU(ID_CHECK_VERSION, CAdvancedFrame::OnCheckVersion)
     EVT_MENU(ID_REPORT_BUG, CAdvancedFrame::OnReportBug)
     EVT_HELP(wxID_ANY, CAdvancedFrame::OnHelp)
     // Custom Events & Timers
@@ -686,6 +686,7 @@
     );
     menuHelp->AppendSeparator();
 
+/*    
     strMenuName.Printf(
         _("Check for new %s version"),
         pSkinAdvanced->GetApplicationShortName().c_str()
@@ -708,6 +709,7 @@
     );
     menuHelp->AppendSeparator();
 
+*/
     strMenuName.Printf(
         _("&About %s..."), 
         pSkinAdvanced->GetApplicationName().c_str()
Index: boinc/clientgui/sg_BoincSimpleFrame.cpp
===================================================================
--- boinc.orig/clientgui/sg_BoincSimpleFrame.cpp
+++ boinc/clientgui/sg_BoincSimpleFrame.cpp
@@ -92,7 +92,7 @@
     EVT_MENU(ID_HELPBOINCMANAGER, CSimpleFrame::OnHelpBOINC)
     EVT_MENU(ID_HELPBOINCWEBSITE, CSimpleFrame::OnHelpBOINC)
     EVT_MENU(wxID_ABOUT, CSimpleFrame::OnHelpAbout)
-    EVT_MENU(ID_CHECK_VERSION, CSimpleFrame::OnCheckVersion)
+    //EVT_MENU(ID_CHECK_VERSION, CSimpleFrame::OnCheckVersion)
     EVT_MENU(ID_REPORT_BUG, CSimpleFrame::OnReportBug)
     EVT_MENU(ID_EVENTLOG, CSimpleFrame::OnEventLog)
     EVT_MOVE(CSimpleFrame::OnMove)
@@ -319,6 +319,7 @@
     );
     menuHelp->AppendSeparator();
 
+/*
     strMenuName.Printf(
         _("Check for new %s version"),
         pSkinAdvanced->GetApplicationShortName().c_str()
@@ -341,6 +342,7 @@
     );
     menuHelp->AppendSeparator();
 
+*/
     strMenuName.Printf(
         _("&About %s..."), 
         pSkinAdvanced->GetApplicationName().c_str()
